Clients: Update bin --help, add mapping for >=36.0 (#420)
rucio/documentation: docs/user/using_the_client.md

Rucio provides several commands for the end-user. See [executables](bin/rucio.md).
The command line client is called ``rucio``.

> Please note that these commands are not meant to be included in python scripts. 
> For such applications, we encourage using the [python client](pathname:///html/site/client.html). 

Reviewer: ```suggestion
Note: for Python scripts, we encourage using the [Python client](pathname:///html/site/client.html) instead of the commands listed here.
```

Files already found locally :                 0
Files that cannot be downloaded :             0
```

# Operator Commands 

Rucio provides a CLI for administrative tasks. The get methods can be executed
by any user, but the set methods require some admin privileges. 

## Account and identity methods

To create a new account:

```bash
  $ rucio account add --type USER --email jdoe@blahblih.com jdoe
```

You can choose different types in the list USER, GROUP, SERVICE. Different
policies/permissions can be set depending on the account type.  Once the account
is created, you need to create and attach an identity to this account:

```bash
  $ rucio account identity add --type X509 \
      --id "CN=jdoe,OU=Users,OU=Organic Units,DC=blih,DC=blah" \
      --email jdoe@blahblih.com \
      --account jdoe
```

The list of possible identity types is X509, GSS, USERPASS, SSH, SAML, OIDC:

```bash
  $ rucio account identity list --account jdoe
  Identity: CN=jdoe,OU=Users,OU=Organic Units,DC=blih,DC=blah,        type: X509
```

You can set attributes to the users:

```bash
  $ rucio account attribute add --key country --value xyz --account jdoe
```

And list these attributes:

```bash

>   $ rucio-admin account attribute list --account jdoe

Reviewer: ```suggestion
  $ rucio account attribute list --account jdoe
```

Reviewer: (not 100% sure about this one)
